client/protocols/analytics.py

```diff
@@ -21,6 +25,70 @@ def run(self, messages):
             # TODO(denero) Get the canonical name of the question
             statistics['question'] = self.args.question
 
+        statistics['started'] = self.check_start(messages['file_contents'])
+        print("DEBUG:" + statistics['started'])
```

Comment:
Is this message intended for students? If not,  you should use `log.info(...)` to record debugging messages.